src/components/Main/HandleInitialState.tsx

```diff
@@ -69,8 +77,8 @@ function HandleInitialState({ location, push, component: Component }: HandleInit
     <Component
       initialState={{
         scenarioState,
-        severityName: DEFAULT_SEVERITY_DISTRIBUTION,
-        severityTable: severityDistribution,
+        severityName,
+        severityTable,
         isDefault: isEqual(scenarioState, defaultScenarioState),
       }}
```

Comment:
This is where the default severity data was loaded unconditionally. Now replaced with the deserialized data.